types/amqp/tslint.json

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
I can remove the no-semis and namespace rule.
However the 'unified signatures' rule would give the incorrect intellisense parameter names.